packages/fxa-auth-server/lib/routes/devices-and-sessions.js

```diff
@@ -588,6 +588,12 @@ module.exports = (
         ) {
           throw new error.featureNotEnabled();
         }
+        
+        // If this request is using a session token we bump the last access time
+        if (credentials.id) {
+          credentials.lastAccessTime = Date.now();
+          await db.touchSessionToken(credentials, {} , true);
```

Comment:
I just checked out the logic in `db.touchSessionToken`. What’s up with the short circuit logic in `if (!this.redis || !features.isLastAccessTimeEnabledForUser(uid))`. It seems like calling this method might not always update the lastAccessTime for some users. Just calling this out incase it’s a contributing factor to the issue reported in bugzilla.

Reply:
I think we can probably remove feature flag logic. On env values have sample rate at 100%.